### PR TITLE
Match RemoveActorDecoration, AddActorDecoration, GetDistanceToGoal

### PR DESCRIPTION
**Pull request by Technoman589**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀 This PR matches 3 functions (+0.01%) on `us`.

File: src/197F40.c [hunk 1]
```diff
@@ -2260,7 +2260,22 @@ ApiStatus GetPlayerActorID(Evt* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "197F40", func_8026E9A0);
 
-INCLUDE_ASM(s32, "197F40", GetDistanceToGoal);
+ApiStatus GetDistanceToGoal(Evt* script, s32 isInitialCall) {
+    s32* args = script->ptrReadPos;
```
Comment (Brotenko) [inline src/197F40.c:2264]:
```suggestion
    Bytecode* args = script->ptrReadPos;
```

File: src/197F40.c [hunk 2]
```diff
@@ -2260,7 +2260,22 @@ ApiStatus GetPlayerActorID(Evt* script, s32 isInitialCall) {
...
+    s32 actorID = evt_get_variable(script, *args++);
+    s32 outVar = *args;
```
Comment (Brotenko) [inline src/197F40.c:2266]: Please use `*args++` if it matches - We always want to advance to the next possible argument, even if it is not used afterwards.

File: src/197F40.c [hunk 3]
```diff
@@ -2272,9 +2287,42 @@ INCLUDE_ASM(s32, "197F40", func_8026ED20);
 
 INCLUDE_ASM(s32, "197F40", func_8026EDE4);
 
-INCLUDE_ASM(s32, "197F40", AddActorDecoration);
+ApiStatus AddActorDecoration(Evt* script, s32 isInitialCall) {
+    s32* args = script->ptrReadPos;
```
Comment (Brotenko) [inline src/197F40.c:2291]:
```suggestion
    Bytecode* args = script->ptrReadPos;
```

File: src/197F40.c [hunk 4]
```diff
@@ -2272,9 +2287,42 @@ INCLUDE_ASM(s32, "197F40", func_8026ED20);
...
 
-INCLUDE_ASM(s32, "197F40", RemoveActorDecoration);
+    actor = get_actor(actorID);
+    actorPart = get_actor_part(actor, partIndex);
+    add_part_decoration(actorPart, decorationIndex, decorationType);
+    return ApiStatus_DONE2;
+}
+
+ApiStatus RemoveActorDecoration(Evt* script, s32 isInitialCall) {
+    s32* args = script->ptrReadPos;
```
Comment (Brotenko) [inline src/197F40.c:2310]:
```suggestion
    Bytecode* args = script->ptrReadPos;
```

**Review by Brotenko — CHANGES_REQUESTED** [review]

**Review by Brotenko — APPROVED** [review]

**Review by nanaian — APPROVED** [review]

**Pull request merged**